src/ReflectionType.php

```diff
@@ -85,14 +85,14 @@ public static function convertToDisplayType(\ReflectionType $type)
             'int'  => 'integer',
             'bool' => 'boolean'
         ];
-        $displayType = $type->type;
+        $displayType = (string)$type;
         if (isset($typeMap[$displayType])) {
             $displayType = $typeMap[$displayType];
         }
 
         $displayType = ltrim($displayType, '\\');
 
-        if ($type->allowsNull()) {
+        if (method_exists($type, 'allowsNull') && $type->allowsNull()) {
```

Comment:
Hmm... I agree that I should revert this code, but for a different reason. I think this is catching the class definition from the polyfill, which had no `$this-&gt;allowsNull()` method. I actually **added** a dummy `$this-&gt;allowsNull()` method to my polly fill, so this should no longer be an issue. I don't particularly like the dummy method in the polyfill, but I think it's the best solution currently. Either way, this code should no longer be necessary. Thanks for pointing this out. 👍